client-go/cmd/keys.go

```diff
@@ -129,7 +128,7 @@ func chooseKey() (api.KeyCreateRequest, error) {
 }
 
 func listKeys() ([]api.KeyCreateRequest, error) {
-	folder := path.Join(os.Getenv("HOME"), ".ssh")
+	folder := path.Join(client.FindHome(), ".ssh")
```

Comment:
I guess we can't use `os.user.Current().HomeDir` since we build this as a static binary.